cmd/helm/install.go

```diff
@@ -462,3 +464,26 @@ func checkDependencies(ch *chart.Chart, reqs *chartutil.Requirements) error {
 	}
 	return nil
 }
+
+//readFile load a file from the local directory or a remote file with a url.
+func readFile(filePath string) ([]byte, error) {
+	u, _ := url.Parse(filePath)
+	scheme := u.Scheme
+	if scheme == "http" || scheme == "https" {
```

Comment:
I wonder if we want to handle other protocols (ftp anybody still uses that? :D) as well, but I guess it should be filed as a separate issue.